Makefile

```diff
@@ -0,0 +1,48 @@
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: start-backend
+start-backend: ## Starts only the backend
+	@docker-compose up -d --build
+	@cd backend/;poetry run ./manage.py runserver
+
+.PHONY: start-frontend-using-backend
+start-frontend-using-backend: ## Start the frontend, using the API
+	@cd frontend/; yarn ember server --proxy=http://localhost:8000
+
+.PHONY: start-frontend
+start-frontend: ## Start the frontend, using Mirage
+	@cd frontend/; yarn ember s
+
+.PHONY: lint-backend
+lint-backend: ## Lint the backend
+	@cd backend/; poetry run black .; poetry run isort .; poetry run flake8; 
```

Comment:
It's good practice to handle failing `cd`, especially with relative paths:

```shell
cd ./somewhere && command
```

This makes sure that, if `cd` failed, the command will not run.

This make target could be rewritten to
```
	@cd ./backend && poetry run black --check .
	@cd ./backend && poetry run flake8
```

Note the `--check` for the black command and the removed `isort`. This is because I don't expect a `lint` make target to change any files.

This remark applies to all the make targets.

Reply:
Done in https://github.com/adfinis/Outdated/pull/12/commits/a8febf1be940b9c0889f786117ebb6a6ed7bcf7b